cyhy/mailer/CyhyMessage.py

```diff
@@ -61,6 +61,8 @@ class CyhyMessage(ReportMessage):
 
 <p>The Cyber Hygiene scan results are attached for your review. Same password as before. (If this is your first report and you have yet to receive a password, please let us know.)</p>
 
+<p><b>Additional details regarding findings, hosts, mitigated vulnerabilities, potentially risky services, etc., can be found under Appendix G: Attachments.</b> To access the attachments embedded within the report, open the report with a dedicated PDF reader (such as Adobe Acrobat), and click on the paper clip icon to the left of the attachment name.
```

Comment:
I'm pretty sure "paperclip" is a single word:
```suggestion
<p><b>Additional details regarding findings, hosts, mitigated vulnerabilities, potentially risky services, etc., can be found under Appendix G: Attachments.</b> To access the attachments embedded within the report, open the report with a dedicated PDF reader (such as Adobe Acrobat), and click on the paperclip icon to the left of the attachment name.
```

Reply:
Looking at the report pdf, the word is spelled paper clip. Do we want to keep it consistent with the report?

Reply:
Yes, I suppose we do.